streams/runners/storm/runtime/src/test/java/com/hortonworks/streamline/streams/runtime/storm/bolt/rules/RulesBoltConditionTest.java

```diff
@@ -161,6 +162,50 @@ public void testProjectNoCondition() throws Exception {
         };
     }
 
+    @Test
+    public void testComplex1() throws Exception {
+        doTest(readFile("/streamline-complex-condition.json"), getWeather("SFO", 0, 0));
```

Comment:
It would be nice if we could add comment on json file to see actual query. It is super hard for human to interpret such rule processor configuration to query.
I'm assuming the query as `select temperature, humidity, city from teststream where temperature + humidity > 100 OR UPPER(city) = 'SFO'` since it is added in other test.

Reply:
will add